### ISSUE TITLE
add framework examples

### ISSUE THREAD

**Issue by danielroe** [unjs/fontaine#7]
It would be good to add some sandboxes or example configurations so it's easy for people to get up-and-running with (at least):

- [ ] Angular
- [ ] Laravel
- [x] Vite
- [x] Next.js

**Comment by johneatmon** [comment]
[Here's my example using Astro](https://stackblitz.com/edit/github-thub7n)

Did I do it right? 😅

**Comment by danielroe** [comment]
@johneatmon That looks perfect! Only issue seems to be that `font-sans` is not used anywhere in the demo, but I think the point is that you've set it up to work when it is used. Would you like to create a PR to add to README?

**Comment by johneatmon** [comment]
Yeah, @danielroe, that's probably something I should add just to make it a little more clear. In Tailwind CSS, sans is automatically applied in the baked-in reset, so it's "implied." I'll make a plain CSS version for Astro as well. 🙂

Would you like both examples on the same PR?

**Comment by johneatmon** [comment]
Here's [a non-Tailwind example of Astro with Fontaine](https://stackblitz.com/edit/github-thub7n-p4mxmm). Also, I updated the Tailwind example with the `font-sans` class applied to the body, to make the example more explicit.

**Comment by danielroe** [comment]
added config example here: https://github.com/danielroe/fontaine/commit/fb0c62beb75b2a2bbc70f632a59acaac69227b82. The Stackblitzes are great - and might work quite well when/if we give fontaine a web site of its own.
